src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -47,28 +47,23 @@ public static class PrereleaseCompatibilityTransformer
 
             switch (version)
             {
-                case "2.0.0-csd.2.beta.2019-01-24":
-                {
-                    // SARIF TC31. Nothing to do.
-                    break;
-                }
-
                 case "2.0.0-csd.2.beta.2019-01-09":
                 {
-                        modifiedLog |= ApplyChangesFromTC31(sarifLog);
-                        break;
+                    // SARIF TC30. Nothing to do.
+                    break;
                 }
 
                 case "2.0.0-csd.2.beta.2018-10-10":
+                case "2.0.0-csd.2.beta.2018-10-10.1":
+                case "2.0.0-csd.2.beta.2018-10-10.2":
```

Comment:
this is wrong. these aren't valid schema references from the JSON. all these versions should refer to the same JSON schema, the point updates are for the SDK backing them (i.e., the semver associated with the nuget package updates).